plugins/de.ovgu.featureide.ui/schema/de.ovgu.featureide.ui.views.exsd

```diff
@@ -0,0 +1,99 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="de.ovgu.featureide.ui" xmlns="http://www.w3.org/2001/XMLSchema">
```

Comment:
Why do we need a new schema? How about the other views in FeatureIDE?

Reply:
We really didn't need to include this schema, and will remove it.

Reply:
I am certain that this schema was necessary for the new extension point (de.ovgu.featureide.ui.views) of the de.ovgu.featureide.ui plugin. This extension point is necessary because the attribute view is part of the de.ovgu.featureide.fm.attributes plugin, unlike the other views.
Without the schema, it still works, but leads to warnings and a missing description of the extension point.